### DISCUSSION TITLE
Option to use the Tools/gen_cubemap.exe instead of Tools/CubeMapGen.exe

### DISCUSSION THREAD

**Discussion by Whurrhurr** [diesel-modding/BeardLib-Editor#671] [Implemented Ideas]
**Is your feature request related to a problem? Please describe.**
ever since commit b2cbca7 the cubelight and cubemap generation tool has been switched to the original OVK tooling version, the problem however with this version is that it does not handle anything above 128x128 well at all. making 512x512 cubelights look horribly blurry (reducing the quality of the light projection texture) and increasing the filesize (from roughly 2,049kb to 6,145kb, [i]nearly triple the filesize for a loss in overall quality[/i] and these take roughly over a minute and a half to just finish compiling 1 cubelight.)

**Describe the solution you'd like**
have an option in World > Environment if you want to use the official OVK cubelight/cubemap generator or the old one (so a toggle that switches between using Tools/CubeMapGen and Tools/gen_cubemap)

**Comment by Luffyyy** [comment]
Adding it, but midway I realized that the big file size is actually dome occlusion being tga instead of compressed dds lol

**Comment by Luffyyy** [comment]
I've added it. Just a note that "blur" on cubemaps is normal and is the same in overkill's cubemaps. Though I made it aware of the resolution chosen.
